README.md

```diff
@@ -687,10 +817,10 @@ While the library supports the core GraphQL features, there are some limitations
 
 ### Fragment Support
 - **Named Fragments**: Not supported due to multiple definition restriction
-- **Inline Fragments**: Not supported (e.g., `... on TypeName`)
+- **Inline Fragments**: ✅ **Fully Supported** (e.g., `... on TypeName`)
```

Comment:
Ultra nit, but again because a clear readme is helpful: Could this please move above "Named fragments" so that we advertise that inline fragments are working at least?